libcaf_openssl/src/openssl/manager.cpp

```diff
@@ -151,6 +152,13 @@ void manager::add_module_options(actor_system_config& cfg) {
       "path to a file of concatenated PEM-formatted certificates")
     .add<std::string>("cipher-list",
                       "colon-separated list of OpenSSL cipher strings to use");
+  // Add the defaults to the config so they show up in --dump-config.
+  auto& grp = put_dictionary(cfg.content, "caf.openssl");
+  put_missing(grp, "certificate", cfg.openssl_certificate);
+  put_missing(grp, "key", cfg.openssl_key);
+  put_missing(grp, "passphrase", cfg.openssl_passphrase);
+  put_missing(grp, "capath", cfg.openssl_capath);
+  put_missing(grp, "cafile", cfg.openssl_cafile);
```

Comment:
The `cipher-list` option has no default value. Just for historic reasons the other options are using member variables. Since the user might write to the member variables, we always have to include them here.